test/k8sT/Services.go

```diff
@@ -818,6 +818,14 @@ var _ = Describe("K8sServicesTest", func() {
 				countOutK8s2, _ = strconv.Atoi(strings.TrimSpace(res.Stdout()))
 			}
 
+			cmdFragmentedPacketsBeforeK8s1 := "cilium bpf metrics list | awk '/Fragmented packet.*INGRESS/ {print $4}'"
```

Comment:
I think this command deserve a func in _test/helpers/wrappers.go_ to avoid its duplication below.